label_studio/core/utils/common.py

```diff
@@ -632,3 +632,16 @@ def collect_versions(force=False):
     return result
 
 
+def get_organization_from_request(request):
```

Comment:
Do we remove this in the future once all sessions will be updated? If so, may be we can leave some TODO message to clear this function in a future release?

Reply:
Yes, we will remove it. Added TODO.